quipucords/tests/integration/conftest.py

```diff
@@ -1,4 +1,13 @@
-"""Common fixtures for integration tests."""
+"""
+Common fixtures for integration tests.
+
+pytest_docker_tools is used to create container images and container fixtures.
+Besides some specifics, it is mostly passing kwargs to docker sdk.
+
+Links to relevant documentation
+- https://github.com/Jc2k/pytest-docker-tools#containers
+- https://docker-py.readthedocs.io/en/stable/containers.html
```

Comment:
Added this here because during development I completely forgot how to play around with pytest-docker-tools